api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java

```diff
@@ -1174,6 +1174,155 @@ public static CohortDefinition getMQ13DenB4_P4(
+    map.put("856", hivViralLoadConcept);
+    map.put("1065", yesConcept);
+    map.put("1982", pregnantConcept);
+
+    String query =
+        "SELECT p.patient_id "
+            + "             FROM patient p "
+            + "             INNER JOIN ( "
+            + "                         SELECT p.patient_id, MIN(e.encounter_datetime) as first_carga_viral  "
+            + "                         FROM patient p  "
+            + "                         INNER JOIN encounter e ON p.patient_id = e.patient_id  "
+            + "                         INNER JOIN obs o ON e.encounter_id = o.encounter_id  "
+            + "                         WHERE  p.voided = 0   "
+            + "                                AND e.voided = 0   "
+            + "                                AND o.voided = 0      "
+            + "                                AND o.concept_id = ${856}   "
+            + "                                AND o.value_numeric >= 50 "
+            + "                                AND ( e.encounter_type = ${6} AND e.encounter_datetime BETWEEN :startDate AND :endDate) "
+            + "                                AND e.location_id = :location   "
+            + "                         GROUP  BY p.patient_id    "
+            + "                       ) AS lab ON lab.patient_id = p.patient_id  "
+            + "             INNER JOIN (  "
+            + "                         SELECT p.patient_id, e.encounter_datetime AS gestante  "
+            + "                         FROM patient p   "
+            + "                         INNER JOIN encounter e ON e.patient_id = p.patient_id   "
+            + "                         INNER JOIN obs o ON e.encounter_id = o.encounter_id   "
+            + "                         INNER JOIN person pe ON p.patient_id = pe.person_id "
+            + "                         WHERE   p.voided = 0   "
+            + "                                 AND e.voided = 0   "
+            + "                                 AND o.voided = 0       "
+            + "                                 AND o.concept_id = ${1982}    "
+            + "                                 AND o.value_coded = ${1065}    "
+            + "                                 AND pe.gender = 'F' "
+            + "                                 AND ( e.encounter_type = ${6} "
+            + "                                 AND e.encounter_datetime BETWEEN :startDate AND :endDate) "
+            + "                                 AND e.location_id = :location    "
+            + "                       ) AS mulher ON mulher.patient_id = p.patient_id  "
+            + "             WHERE p.voided = 0  "
+            + "                   AND lab.first_carga_viral = mulher.gestante";
+
+    StringSubstitutor stringSubstitutor = new StringSubstitutor(map);
+
+    sqlCohortDefinition.setQuery(stringSubstitutor.replace(query));
+
+    return sqlCohortDefinition;
+  }
```

Comment:
This query already exists, please remove and use the other one.

Reply:
Done